library/src/androidTest/java/com/owncloud/android/FileIT.java

```diff
@@ -162,6 +163,8 @@ public void testShareViaLinkSharees() {
 
     @Test
     public void testShareToGroupSharees() {
+        onlyOnMin(OwnCloudVersion.nextcloud_17);
```

Comment:
Should be in `@BeforeClass` instead, since it's added to all methods in this class.

Reply:
First 5 tests do not have it, so we cannot add it to tall.

Reply:
Oh yeah, completely ignored the first 5 for some reason. Sorry!